students/leejiyub/users/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
+class User(models.Model):
+    name = models.CharField(max_length=20)
+    email = models.CharField(max_length=100)
+    password = models.CharField(max_length=256)
+    phonenumber = models.IntegerField()
+    gender = models.CharField(max_length=10)
```

Comment:
```suggestion
    name        = models.CharField(max_length=20)
    email       = models.CharField(max_length=100)
    password    = models.CharField(max_length=256)
    phonenumber = models.CharField(max_length=100)
    gender      = models.CharField(max_length=10)
```
1. = 을 기준으로 정렬해주세요. 위코드 컨벤션입니다.
2. 전화번호 같은 겨우 문자가 포함 될수 있습니다. (- or +) 따라서 charfield로 바꿔 주세요.
3. 데이터의 이력 관리를 위해서 created_at, updated_at column을 추가해주세요.

```python
created_at = models.DateTimeField(auto_now_add=True)
updated_at = models.DateTimeField(auto_now=True)
```

Reply:
수정했습니다 확인 부탁드립니다.